run-lamp.sh

```diff
@@ -28,6 +28,13 @@ fi
 # enable php short tags:
 /bin/sed -i "s/short_open_tag\ \=\ Off/short_open_tag\ \=\ On/g" /etc/php/7.0/apache2/php.ini
 
+# display PHP errors:
+if [ $DISPLAY_ERRORS ]; then
+    /bin/sed -i "s/display_errors\ \=\ Off/display_errors\ \=\ On/g" /etc/php/7.0/apache2/php.ini
```

Comment:
This would enable  `display_errors` even if `DISPLAY_ERRORS=no`. We should use `exportBoolean` for this instead.